playwright_test/locaterPackage/kmLocators.js

```diff
@@ -0,0 +1,22 @@
+
+/*
+ * kmLocators.js
+ * playwright_test/utils
+ * Created by Archit on 21/12/2011. 
+ */
+
+export const locators = {
+    envBtn : '//label[@for="test"]',
+    appIdField : '//span[text()="App ID"]/following::input',
+    scriptFiled : '//textarea[@data-gramm="false"]',
+    launchWidgetBtn : '(//button[@type="button"])[3]',
+    logoutWidgetBtn : '//button[text()="Logout & Reload"]'
+}
+
+export const widgetLocators = {
+    kmIframe : 'iframe[name="Kommunicate widget iframe"]',
+    kmLaunchWidget : '//*[@id="launcher-svg-container"]',
```

Comment:
for locating the element using id you can simply use '#'

Reply:
I guess it doesn't matter ???

Reply:
please change it

Reply:
is it going to cause any issues in the future ??

Reply:
@archit2602  Locating with ID is faster than XPath. Sometimes XPath will not work on some browsers
https://qr.ae/prtsgL

Reply:
#1002 fixed all the locator-related issues in this PR @vipinsonu1 @abhishek7214